packages/@aws-cdk/aws-ecs-patterns/lib/fargate/load-balanced-fargate-service.ts

```diff
@@ -72,6 +73,27 @@ export interface LoadBalancedFargateServiceProps extends LoadBalancedServiceBase
    * @default true
    */
   readonly createLogs?: boolean;
+
+  /**
+   * Override for the Fargate Task Definition execution role
+   *
+   * @default - No value
```

Comment:
Describe the default *behavior*, not the default value

Reply:
Fixed